src/s2e/Plugins/ExecutionMonitors/LibraryCallMonitor.cpp

```diff
@@ -65,7 +65,7 @@ void LibraryCallMonitor::onIndirectCallOrJump(S2EExecutionState *state, uint64_t
 
     // Only interested in particular modules loaded in the process
     const ModuleDescriptor *currentMod = m_map->getModule(state, pc);
-    if (!(m_monitorAllModules || (currentMod && m_procDetector->isTracked(currentMod->Name)))) {
+    if (!(m_monitorAllModules || (currentMod && m_procDetector->isTracked(state)))) {
```

Comment:
I think this makes sense in general.

Reply:
Yeah, agreed. Not sure why I did it the other way in the first place...